app/viewmodels/helpers/events/MovementEventHelper.scala

```diff
@@ -459,4 +460,46 @@ class MovementEventHelper @Inject()(
       )
     )
   }
+
+  def ie829AcceptedExportDetails(implicit notificationOfAcceptedExport: NotificationOfAcceptedExportModel, messages: Messages): Html = {
+    def acceptedExportSummary: Html = {
+      val acceptedDate = summaryListRowBuilder("movementCreatedView.section.ie829.acceptedDate", notificationOfAcceptedExport.dateOfAcceptance.formatDateForUIOutput())
+      val senderCustomsOfficeReference = summaryListRowBuilder("movementCreatedView.section.ie829.senderCustomsOfficeReference", notificationOfAcceptedExport.referenceNumberOfSenderCustomsOffice)
+      val senderCustomsOfficer = summaryListRowBuilder("movementCreatedView.section.ie829.senderCustomsOfficer", notificationOfAcceptedExport.identificationOfSenderCustomsOfficer)
+      val documentReferenceNumber = summaryListRowBuilder("movementCreatedView.section.ie829.documentReferenceNumber", notificationOfAcceptedExport.documentReferenceNumber)
+
+      buildOverviewPartial(
+        summaryListAttributes = Map("id" -> "accepted-export-summary"),
+        summaryListRows = Seq(
+          Some(acceptedDate),
+          Some(senderCustomsOfficeReference),
+          Some(senderCustomsOfficer),
+          Some(documentReferenceNumber)
+        )
+      )
+    }
+
+    def consigneeSummary: Html = {
+      val name = summaryListRowBuilder("movementCreatedView.section.consignee.name", notificationOfAcceptedExport.consigneeTrader.traderName.getOrElse(""))
+      val vat = summaryListRowBuilder("movementCreatedView.section.consignee.vatNumber", notificationOfAcceptedExport.consigneeTrader.traderExciseNumber.getOrElse(""))
+      val address = notificationOfAcceptedExport.consigneeTrader.address.map(address => summaryListRowBuilder("movementCreatedView.section.consignee.address", renderAddress(address)))
+      val eori = summaryListRowBuilder("movementCreatedView.section.consignee.eoriNumber", notificationOfAcceptedExport.consigneeTrader.eoriNumber.getOrElse(""))
+
+      buildOverviewPartial(
+        headingTitle = Some("movementCreatedView.section.ie829.consignee.heading"),
+        headingId = Some("consignee-information-heading"),
+        summaryListRows = Seq(
+          Some(name),
+          Some(vat),
+          address,
+          Some(eori)
+        ),
+        summaryListAttributes = Map("id" -> "consignee-information-summary")
+      )
+    }
+
+    HtmlFormat.fill(
+      Seq(acceptedExportSummary, consigneeSummary)
+    )
+  }
```

Comment:
Rather than this, can't this just use the existing `consigneeInformationCard`?

I don't see why the `consignee` information from the IE829 would be any different from what's against the IE801?

Reply:
I thought for an export we are going to end up with the VAT number in the `traderId` field so if I was to use `consigneeInformationCard` then we would end up showing the label as ERN rather than Identification number

Reply:
But that would also need fixing in the `consigneeInformationCard`

Reply:
There's an existing bug with that, which I have a feeling Ladon might be fixing in his branch.

Reply:
I suppose I've done it like this as the IE801 event would need to check the movement type to work out if it should show `Excise Registration Number (ERN)` , `Identification number for temporary registered consignee` or `Identification number`. Whereas the IE829 is only ever going to come through for an export so we will have a VAT number in the `consignee.traderid` field with the label of `Identification number`

Reply:
@MJCallahanPage refactored to use `consigneeInformationCard ` at https://github.com/hmrc/emcs-tfe-frontend/pull/167/commits/632a57981ee7f41b41d410e1661c7188f3ce1baa